.golangci.yml

```diff
@@ -10,7 +10,7 @@ run:
   issues-exit-code: 1
   tests: true
   build-tags: [""]
-  skip-dirs:  [""]
+  skip-dirs:  ["", "pkg/doc/bbs/bbs12381g2pub/internal/kilic/bls12-381"]
```

Comment:
not sure why we need an empty "" as the first element?

Reply:
I don't know as well why we had it before. Just left it as is to be safe against any unexpected regression.

Reply:
Please remove the "" and reformat as a list.

Reply:
Done